openshift-ci/README.md

```diff
@@ -33,4 +33,27 @@ After making a change to the base image configuration in your repo, trigger a ne
 
 . Navigating to [openshift ci](https://api.ci.openshift.org/console/project/integr8ly/browse/builds)
 . Choosing the relevant image
-. Clicking on the `Start Build` button
\ No newline at end of file
+. Clicking on the `Start Build` button
+
+
+## Image Mirroring for built integreatly images
+
+For information on image-mirroring (aka pushing images to quay after building images and running prow test(s)) see [this section](https://github.com/openshift/release/tree/master/core-services/image-mirroring) in the openshift release documentation.
+
+We've already done [some setup](https://github.com/openshift/release/pull/6766) to add the heimdall-operator to image-mirroring. To add your repo add an additional mapping file to [this folder](https://github.com/openshift/release/tree/master/core-services/image-mirroring/integr8ly) based on openshift release documtenation linked above.
+
+### Credentials
+
+To add your repo, update integreatly+prow robot to have read/write access to the repo.
+
+You shouldn’t need to add further credentials as the [integreatly+prow robot](https://quay.io/organization/integreatly?tab=robots)’s credentials are already sent to the DPTP team for [inclusion in ci](https://github.com/openshift/release/pull/6773/files).
+
+If for some reason you do need to add credentials for another account/bot, here are the steps:
+
+Ask @nmoraitis in the #forum-testplatform channel on slack for his public gpg key and tell him that you need credentials added.
```

Comment:
Probably better not mention individuals here, pinging #test-platform should be enough to get someone's gpg key i would have thought